src/alexapi/device_platforms/orangepi.py

```diff
@@ -18,10 +19,10 @@ def __init__(self, config):
 		self.__pconfig['lights'] = [self.__pconfig['rec_light'], self.__pconfig['plb_light']]
 
 		self.long_press_setup = False
-		if ('long_press' in self.__pconfig
-				and 'command' in self.__pconfig['long_press']
-				and len(self.__pconfig['long_press']['command']) > 0
-				and 'duration' in self.__pconfig['long_press']):
+		if ('long_press' in self.__pconfig and
```

Comment:
So I'm not sure I like this. My way, each additional condition is responsible for adding _and_ and I can take out every single one of them without affecting other lines. The _and_-at-the-end-of-the-line way seems silly this way. This is in other places too.